recipe/meta.yaml

```diff
@@ -10,18 +10,20 @@ source:
   sha256: f271b298b97f5955d53fb12b72c1fb1948c22c1a6b70b315c54cedaca0264ef5
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<38]
```

Comment:
Upstream supports for python 3.7 and above. 
https://github.com/pypa/pyproject-hooks/blob/v1.0.0/pyproject.toml#L11

Reply:
Fixed!